wrap.js

```diff
@@ -112,7 +114,14 @@ module.exports = function wrap (sv, flume) {
       else sv.close(err, cb)
     },
     meta: meta,
-    destroy: sv.destroy
+    destroy: (cb) => {
+      sv.destroy((err) => {
+        cb(err)
+      })
+    },
+    setDestroying: val => { isDestroying = val },
+    isDestroying: () => isDestroying,
```

Comment:
hey @christianbundy I've made this change. It avoids the "setting / getting magic properties".
Not sure it's the best match with the code style here but I feel more secure :hugs: